hash/nes.xml

```diff
@@ -43154,6 +43812,26 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="zenbei" cloneof="allprobb">
+		<description>Zenbei!! Pro Basket (Jpn)</description>
+		<year>1989</year>
+		<publisher>Vic Tokai</publisher>
+		<info name="serial" value="VIC-A2"/>
+		<info name="release" value="19890721"/>
+		<info name="alt_title" value="全米プロバスケット"/>
```

Comment:
As far as I can tell, the Japanese title screen has the “‼” in the title.

Reply:
Yes it does, but the box and cart do not have the two exclamation points. JP Wikipedia's list of games, store pages, and most Youtube videos I saw also exclude them.

Reply:
Why didn’t you use the double exclamation mark character?  (It’s 1-8-75 in JIS X 0213, or U+203C).